testhelpers/org.eclipse.n4js.cli.tests.helper/src/org/eclipse/n4js/cli/runner/helper/NodejsRunner.java

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright (c) 2016 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+package org.eclipse.n4js.cli.runner.helper;
+
+import static com.google.common.base.CharMatcher.breakingWhitespace;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.n4js.binaries.IllegalBinaryStateException;
+import org.eclipse.n4js.binaries.nodejs.NodeJsBinary;
+import org.eclipse.n4js.runner.RunConfiguration;
+import org.eclipse.xtext.xbase.lib.Exceptions;
+
+import com.google.common.base.Joiner;
+import com.google.common.base.Splitter;
+
+/**
+ * Concrete runner, i.e. runner implementation for node.js engine.
+ */
+public class NodejsRunner {
```

Comment:
This looks like a lot of copied code. Can you maybe just call `org.eclipse.n4js.runner.nodejs.NodeRunner`?